examples/tensorflow/classification/main.py

```diff
@@ -101,6 +101,18 @@ def get_dataset_builders(config, num_devices, one_hot=True):
     return [train_builder, val_builder]
 
 
+def get_num_classes(dataset):
+    if 'imagenet' in dataset:
+        num_classes = 1000
+    elif dataset == 'cifar100':
+        num_classes = 100
+    elif dataset == 'cifar10':
+        num_classes = 10
+    else:
+        raise RuntimeError('Could not determine number of classes by dataset name')
```

Comment:
Please return the default value = 1000 and info message how many class classification problem is solved.